src/concrete/ml/quantization/base_quantized_op.py

```diff
@@ -720,12 +725,40 @@ def calibrate(self, *inputs: numpy.ndarray) -> numpy.ndarray:
         if isinstance(raw_result, RawOpOutput):
             return raw_result
 
-        quantized_samples = QuantizedArray(self.n_bits, raw_result)
+        supports_analytical_quant = all(
+            isinstance(qv, QuantizedArray) for qv in inputs
+        ) and isinstance(self, QuantizedMixingOp)
```

Comment:
Shouldn't this be in QuantizedMixingOp?

Reply:
yes, it could be a base function that returns False and an overload in the QuantizedMixingOp that implements the condition